docs/modules/ROOT/pages/gsn.adoc

```diff
@@ -0,0 +1,93 @@
+= The Gas Station Network
+
+User onboarding is one of the hottest topics in Ethereum, and meta-transactions the de-facto solution to this important problem. Here you'll learn all about the Gas Station Network (GSN) and how to write contracts that don't require their users to hold Ether to pay for gas.
+
+If you're already up to speed with the workings of the GSN, feel free to skip to <<Receiving a relayed call>>, where we'll go over how to use the OpenZeppelin Contracts to easily write a GSNRecipient contract. Otherwise, strap in!
+
+== Sending gas-less transactions
+
+All Ethereum transactions use gas, and the sender of each transaction must have enough Ether to pay for the gas spent. Even though these gas costs are low for basic transactions (a couple cents), getting this Ether is no easy task: dApp users often need to go through Know Your Customer and Anti Money-Laundering processes (KYC & AML), which not only take time but often involve sending a selfie holding their passport over the Internet (!). Only the most hardcore users will put up with this hassle, and dApp adoption greatly suffers when Ether is required. We can do better.
```

Comment:
"a couple cents" could be "a couple of cents"

"which not only take time but often involve sending a selfie holding their passport over the Internet (!)" could be
"which not only takes time but often involves sending a selfie holding their passport over the Internet(!)"